include/net/icalendar_parser.rst

```diff
@@ -0,0 +1,47 @@
+.. _icalendar_parser_readme:
+
+iCalendar parser
+################
+
+`iCalendar`_ is data format for representing and exchanging calendaring and scheduling information. The iCalendar parser library can be used to parse a shared iCalendar at specified URL (HTTP or HTTPS) and returns paresd calendar events. It supports IPv4 and IPv6 protocols.
```

Comment:
I am a bit surprised to see information about IP protocols here. IMO the parser should be able to parse an ical "file" from a buffer provided by the user. We have other libraries to download files using HTTP for instance, if this is the use case we want to show in the sample application. What is your opinion?